packages/hash/frontend/src/components/layout/PageSidebar/WorkspaceSwitcher.tsx

```diff
@@ -0,0 +1,297 @@
+                    outlineOffset: "2px",
+                  },
+                }}
+              >
+                <Avatar
+                  size={38}
+                  sx={{
+                    mr: 0.75,
+                  }}
+                  title={org.properties.name}
+                />
+                <Box
+                  sx={{
+                    display: "flex",
+                    flexDirection: "column",
+                  }}
+                >
+                  <Typography
+                    variant="smallTextLabels"
+                    sx={{
+                      mb: "2px",
+                    }}
+                  >
+                    {org.properties.name}
+                  </Typography>
+                  <Typography
+                    variant="microText"
+                    sx={{
+                      color: ({ palette }) => palette.gray[50],
+                      fontWeight: 500,
+                    }}
+                  >{`${org.memberships.length} members`}</Typography>
+                </Box>
+              </Link>
+            ))}
+          </Box>
+
+          <Box
+            sx={{
+              px: 0.5,
+              py: 0.75,
+              display: "flex",
+              flexDirection: "column",
+            }}
+          >
+            {[
+              {
+                title: "Workspace Settings",
+                id: 1,
+                href: "/",
+              },
+              {
+                title: "Create or Join a workspace",
+                id: 2,
+                href: "/",
+              },
+            ].map(({ title, id, href }) => (
+              <Link
+                key={id}
+                href={href}
+                noLinkStyle
+                sx={{
+                  px: 1.5,
+                  py: 1,
+                  borderRadius: "4px",
+                  //   @todo-mui this is a default style and should be placed in the Link component
+                  "&:hover": {
+                    backgroundColor: ({ palette }) => palette.gray[20],
+                  },
+
+                  "&:focus-visible": {
+                    outline: ({ palette }) => `2px solid ${palette.blue[70]}`,
+                    outlineOffset: "2px",
+                  },
+                }}
+              >
+                <Typography
+                  variant="smallTextLabels"
+                  sx={{
+                    lineHeight: 1,
+                    color: ({ palette }) => palette.gray[80],
+                    fontWeight: 500,
+                  }}
+                >
+                  {title}
+                </Typography>
+              </Link>
+            ))}
+          </Box>
+
+          <Box
+            sx={{
+              p: "4px 4px 6px 4px",
+              borderTop: ({ palette }) => `1px solid ${palette.gray[30]}`,
+            }}
+          >
+            {/* @todo use the LinkButton for this once merged in */}
+            <Box
```

Comment:
I think this should be a MUI `List` where the items are `ListItemButton`s. This would match the implementation of the `AccountDropdown` in the header.

Also currently the spacing here is too large (would like be fixed by using `ListItemButton`):
<img width="285" alt="image" src="https://user-images.githubusercontent.com/42802102/156802205-3cf537c0-9b6a-407c-91b7-0e78b94ddb3c.png">

Reply:
Thanks for catching. I was talking with @akash-joshi earlier and mentioned we should probably be using Menu/MenuItem for UIs like this as opposed to Popover/(Box, ListItemButton) since they are technically menus and Menu/MenuItem handle the accessibility features properly

Reply:
Plan to replace existing implementations with Menu/MenuItem in this task => [internal link](https://app.asana.com/0/1201864704093982/1201926880408096)